modules/rostests/winetests/kernel32/path.c

```diff
@@ -2022,6 +2026,19 @@ if (0)
     ret = pDeactivateActCtx(0, cookie);
     ok(ret, "failed to deactivate context, %u\n", GetLastError());
     pReleaseActCtx(handle);
+
+    /*
+       Test buffer overflow.
+       lenof(longDirName) + lenof(longFileName) + lenof(ext) = MAX_PATH
+    */
+    len = pSearchPathW(longDirName,
```

Comment:
Note also that here you're testing kernel32!SearchPathW, and not directly ntdll!RtlDosSearchPath_Ustr, so I could say that you may not be 100% certain that what you're testing is genuinely exercising RtlDosSearchPath_Ustr.

Reply:
You are right.
rostests\apitests\ntdll\RtlDosSearchPath_Ustr.c better place for this test.